node-sdk-example/test/createSandbox.test.js

```diff
@@ -1,7 +1,10 @@
 import {
     ApiClient,
     CreateSandboxRequest,
+    CustomPatch,
```

Comment:
Are these unused? If so, let's delete them.

Reply:
@Freezerburn PTAL at this one.